imjoy/worker_template.py

```diff
@@ -556,36 +488,24 @@ def main():
 
     opt = parser.parse_args()
 
+    # TODO: We can probably remove this since the script directory will be first in sys.path.
     if "" not in sys.path:
         sys.path.insert(0, "")
 
     imjoy_path = os.path.dirname(os.path.normpath(__file__))
     if imjoy_path not in sys.path:
         sys.path.insert(0, imjoy_path)
 
+    # ENDTODO
```

Comment:
Can we remove this part?

Reply:
We add this to support that one can run `imjoy` in a folder and the python modules inside can be imported directly.

For the second one, maybe to just support importing the sockeio client. Now I think we don't need it since we are going to replace the socketio package.

Maybe these are not useful after all, I think we can remove these two.

Reply:
I think this is already resolved by the default python behavior.

https://chrisyeh96.github.io/2017/08/08/definitive-guide-python-imports.html

> The directory containing the script being run is placed at the beginning of the search path, ahead of the standard library path.